content/pt-br/includes/task-tutorial-prereqs.md

```diff
@@ -1,6 +1,3 @@
-Você precisa de um cluster Kubernetes e a ferramenta de linha de comando kubectl 
-precisa estar configurada para acessar o seu cluster. Se você ainda não tem um
-cluster, pode criar um usando o [minikube](/docs/tasks/tools/#minikube)
-ou você pode usar um dos seguintes ambientes:
+Você precisa ter um cluster do Kubernetes e a ferramenta de linha de comando kubectl deve estar configurada para se comunicar com seu cluster. É recomendado executar esse tutorial em um cluster com pelo menos dois nós que não estejam atuando como hosts de plano de controle. Se você ainda não possui um cluster, pode criar um usando o [minikube](/docs/tasks/tools/#minikube) ou pode usar um desses playgrounds do Kubernetes:
```

Comment:
Two bits here:
* We standardized the translation of control plane as "camada de gerenciamento" instead of "plano de controle" in the Portuguese localization. Could you please update this term?
* I'd prefer if the last sentence was reverted to the original ("ou pode usar um dos seguintes ambientes").

Reply:
Thank you for the review! 
I implemented the points you bought up.
thanks :)